src/snmpd/patch-5.9.3+dfsg/series

```diff
@@ -0,0 +1,4 @@
+0008-Enable-macro-DEB_BUILD_ARCH_OS-in-order-to-build-ipv.patch  
```

Comment:
I see `0001-SNMP-Stop-spamming-logs-with-statfs-permission-denie.patch`, `0009-Makefile.in-agent-Makefile.in-Fix-parallel-compilati.patch`, `0010-Makefile.in-Make-sure-that-sedscript-is-built-before.patch`, `0011-agent-Makefile.in-Build-the-MIB-module-code-once.patch`, and `0012-agent-Makefile.in-Unbreak-the-enable-minimalist-buil.patch` were dropped. But it doesn't seem like the changes in those patches are in 5.9.3. Is there a reason these patches were dropped?

Reply:
`0001-SNMP-Stop-spamming-logs-with-statfs-permission-denie.patch` is needed, `0008-Enable-macro-DEB_BUILD_ARCH_OS-in-order-to-build-ipv.patch` can be optionally dropped (since that logic is now coming from `/usr/share/dpkg/architecture.mk`), `0012-agent-Makefile.in-Unbreak-the-enable-minimalist-buil.patch` is needed. The others are correct.

Reply:
Good point. Based on what's in sonic-net/sonic-snmpagent#22, there might not be any changes needed at all. For now, I prefer to let this get merged in, and then look at removing the snmp build (either in time for 202405 release or after the branch cutoff).